CHANGELOG.rst

```diff
@@ -4,6 +4,64 @@ Regolith Change Log
 
 .. current developments
 
+v0.2.0
+====================
```

Comment:
hmmm did you run rever on your branch or something?  That probably should be reverted.  You can use `rever undo` if the `rever/` dir is still around

Reply:
Actually @CJ-Wright points out this might be from a rebase issue.